app/src/app/api/v0/inventory/[inventory]/route.ts

```diff
@@ -15,6 +18,24 @@ export const GET = apiHandler(async (req, { params }) => {
     throw new createHttpError.NotFound("Inventory not found");
   }
 
+  // @ts-ignore
+  const { totalEmissions } = await ActivityValue.findOne({
```

Comment:
Can you add a TODO here that we run this calculation at save time and store it in the Inventory.totalEmissions field?

Reply:
https://openearth.atlassian.net/browse/ON-2429